OrbitCaptureClient/CaptureClient.cpp

```diff
@@ -40,18 +40,13 @@ void CaptureClient::Capture(
   }
   capture_options->set_trace_gpu_driver(true);
   for (const auto& pair : selected_functions) {
-    const FunctionInfo* function = pair.second;
-    // TODO: this is temporary fix. We should understand why in
-    // Capture::GSelectedFunctionsMap could be nullptrs and fix it accordingly
-    if (function == nullptr) {
```

Comment:
If I remember correctly, there are a couple of other places where we check function for nullptr. I guess we should also remove them (maybe in a different PR).